src/scenes/AcceleratorRouter/Applications/ApplicationReviewModal.tsx

```diff
@@ -0,0 +1,183 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+
+import { Grid, useTheme } from '@mui/material';
+import { Dropdown, DropdownItem } from '@terraware/web-components';
+
+import DialogBox from 'src/components/common/DialogBox/DialogBox';
+import TextField from 'src/components/common/Textfield/Textfield';
+import Button from 'src/components/common/button/Button';
+import { requestReviewApplication } from 'src/redux/features/application/applicationAsyncThunks';
+import { selectApplicationReview } from 'src/redux/features/application/applicationSelectors';
+import { useAppDispatch, useAppSelector } from 'src/redux/store';
+import strings from 'src/strings';
+import { Application, ApplicationReview, ApplicationReviewStatus, ApplicationStatus } from 'src/types/Application';
+import useForm from 'src/utils/useForm';
+
+export type ApplicationReviewModalProps = {
+  open: boolean;
+  onClose: () => void;
+  onSuccess: () => void;
+  application: Application;
+};
+
+const ApplicationReviewModal = ({
+  open,
+  onClose,
+  onSuccess,
+  application,
+}: ApplicationReviewModalProps): JSX.Element => {
+  const theme = useTheme();
+
+  const dispatch = useAppDispatch();
+
+  const [requestId, setRequestId] = useState<string>('');
+  const result = useAppSelector(selectApplicationReview(requestId));
+
+  const statusOptions: ApplicationReviewStatus[] = [
```

Comment:
This could be moved to src/types/Application, like here:
https://github.com/terraware/terraware-web/blob/main/src/types/Deliverables.ts#L21